src/cell.rs

```diff
@@ -201,13 +196,13 @@ impl Cell {
     }
 
     pub fn config(&self) -> CellConfig {
```

Comment:
`fn config` should check whether stage 1 translation in el2 is enabled. Otherwise it will get an fault address which leads to issue #8